sdk/tables/azure-data-tables/README.md

```diff
@@ -313,7 +314,7 @@ Get started with our [Table samples][tables_samples].
 Several Azure Tables Python SDK samples are available to you in the SDK's GitHub repository. These samples provide example code for additional scenarios commonly encountered while working with Tables.
 
 ### Common Scenarios
-These code samples show common scenario operations with the Azure Tables client library. The async versions of the samples (the python sample files appended with _async) show asynchronous operations with Tables and require Python 3.6 or later.
+These code samples show common scenario operations with the Azure Tables client library. The async versions of the samples (the python sample files appended with _async) show asynchronous operations with Tables and require Python 3.7 or later.
```

Comment:
To make future maintenance easier, we can remove the bit about requiring a certain version of Python for async samples. Since we dropped 2.7 support, users will always be on a version of Python that support async operations.
```suggestion
These code samples show common scenario operations with the Azure Tables client library. The async versions of the samples (the python sample files appended with _async) show asynchronous operations.
```